website/static/js/nodesDelete.js

```diff
@@ -145,34 +145,31 @@ var NodesDeleteViewModel = function(node) {
     });
 
     self.message = ko.computed(function() {
-        if (self.page() === self.CONFIRM) {
-            if (self.nodeType === 'project')
-                return 'The following project and components will be deleted';
+        var message = 'It looks like your ' + self.nodeType + ' has components within it. To delete this ' +
+          self.nodeType + ', you must also delete all child components.';
 
-            if (self.nodeType === 'component')
-                return 'The following components will be deleted';
-        }
+        var preprintMessage = '<br><br>This ' + self.nodeType + ' also contains a <strong>preprint</strong>. Deleting this ' +
+          self.nodeType + ' will also delete your <strong>preprint</strong>. This action is irreversible.';
 
-        if (self.nodeType === 'project')
-            return {
-                select: 'It looks like your project has components within it. To delete this project, you must also delete all child components',
-                confirm: 'The following project and components will be deleted.'
-            }[self.page()];
+        var confirm_message = 'The following ' + (self.nodeType === 'project' ? 'project and ' : '') + 'components will be deleted.';
 
-        if (self.nodeType === 'component')
-            return {
-                select: 'It looks like your componet has components within it. To delete this component, you must also delete all child components',
-                confirm: 'The following components will be deleted.'
-            }[self.page()];
+        if (self.page() === self.CONFIRM) {
+            return confirm_message;
+        }
+
+        return {
+            select: message + (self.isPreprint ? preprintMessage : ''),
```

Comment:
I know we have a lot of messages here, so maybe this is a product question, but in the QuickDelete case, this is one of the messages, "It will no longer be available to other contributors on the project." Is this also needed in the `NodesDeleteViewModel`.